newrelic/hooks/framework_django.py

```diff
@@ -445,8 +445,8 @@ def _nr_wrapper_BaseHandler_get_response_(wrapped, instance, args, kwargs):
     request = _bind_get_response(*args, **kwargs)
 
     if hasattr(request, '_nr_exc_info'):
-        if not ignore_status_code(response.status_code):
-            record_exception(*request._nr_exc_info)
+        if not should_ignore_error(request._nr_exc_info, response.status_code):
+            notice_error(request._nr_exc_info)
```

Comment:
The if statement is unnecessary, you can pass response.status_code into notice_error and let it handle evaluating.